src/dispatch/incident/scheduled.py

```diff
@@ -110,6 +111,12 @@ def daily_summary(db_session=None):
                                     f"*Incident Commander*: <{incident.commander.weblink}|{incident.commander.name}>"
                                 ),
                             },
+                            "block_id": f"{ConversationButtonActions.invite_user}-{idx}",
+                            "accessory": {
+                                "type": "button",
+                                "text": {"type": "plain_text", "text": "Get Involved"},
+                                "value": f"{incident.id}",
```

Comment:
Does this need to be an f-string?

Reply:
I wasn't confident if we passed an int, slack would do the right thing. So I just forced it to be a string to be safe.

Reply:
Ok